pages/coronavirus-chatbot.md

```diff
@@ -6,7 +6,7 @@ hidesidenav: true
 
 # The title of the tab.
 title: VA coronavirus chatbot
-botframework_cdn: https://cdn.botframework.com/botframework-webchat/4.8.1/webchat-es5.gzip.js
+botframework_cdn: https://cdn.botframework.com/botframework-webchat/4.9.0/webchat-es5.js
```

Comment:
If there are vets-website changes that are dependent on this version bump, try to merge the PRs at the same time before the daily deployment to make sure they go out at the same time

Reply:
Thanks! This version bump wasn't related to any vets-website changes, but we'll keep that in mind.